vendor.conf

```diff
@@ -114,7 +114,7 @@ github.com/googleapis/gax-go v2.0.0
 google.golang.org/genproto 694d95ba50e67b2e363f3483057db5d4910c18f9
 
 # containerd
-github.com/containerd/containerd v1.2.0-beta.0
+github.com/containerd/containerd 3f42445e38d1081f4b8c3b8d7d1ed1860198ed7a
```

Comment:
ping @dmcgowan should the upstream change be included in the 1.2.x branch?

Reply:
The 1.2.x branch won't get created until first rc, this is fine

Reply:
Ah, perfect; thanks!